editor/plugins/node_3d_editor_plugin.h

```diff
@@ -269,6 +269,7 @@ class Node3DEditorViewport : public Control {
 	ObjectID clicked;
 	Vector<_RayResult> selection_results;
 	bool clicked_wants_append;
+	bool selection_in_progress;
```

Comment:
This should be given a default value (likely `false`).

Reply:
Done. I hope it's fine in the header. I heard you are switching away from initializer lists/constructors for that.